stance/main.py

```diff
@@ -9,47 +11,54 @@
 def main():
     # Example calling: python main.py -v etl/squat.mp4
     parser = argparse.ArgumentParser()
-    parser.add_argument("-v", "--motion_video", help="filepath to video containing the user performing a motion")
+    parser.add_argument("-f", "--video_path", help="filepath to video containing the user performing a motion")
+    parser.add_argument("-v", "--verbose", help="Verbosity level. 1 (v) displays info, 2 (vv) displays debug logs",
+                        action="count")
     parser.add_argument("-p", "--use_pickle", help="uses the pickle file corresponding to the video file given",
                         action="store_true")
     args = parser.parse_args()
 
+    verbosity = args.verbose or 0
+    logger.setLevel(logging.WARNING - (10 * verbosity))
     logger.info("Starting Motion")
+
     # Set the motion to be Squat
-    motion = Squat()
+    motion = Squat(verbose=verbosity)
 
     # Find the user skeletons throughout the video
     logger.info("Fitting Video")
```

Comment:
Can you do a timed log for this one too? This one would take the longest, and it would be good to know how long the whole process took.

Reply:
This one already does time internally